### ISSUE TITLE
a11y P2 - Voice control (iOS) - Brand bug

### ISSUE THREAD

**Issue by HarveyPeachey** [bbc/simorgh#5514]
**Describe the bug**
The BBC Brand link in the banner does not work when you say 'Tap BBC News Pidgin' on https://www.bbc.com/pidgin When you say 'Show names' and 'Show numbers' the brand link is also not displayed. Note, not a P1 as you can access the home page via the 'Home' link in the navigation. 

I've tested removing the role text attribute and the lang attribute from the visually hidden link text and this appears not to affect it. This appears to be a bug related to the visually hidden CSS. Though note, you can say 'tap video' and this works for promos with videos on the url and this is visually hidden text, maybe that's because this element also has displayed text with it? I've tested adding some displayed text to the brand link and this fixes it, though not a solution we can use.

Also tested using a SVG title element instead of VH text, this resulted in the brand link being available when you say 'show numbers' but not the other navigation methods listed.

Note the brand logo works on a News Optimo article https://www.bbc.co.uk/news/articles/clldg965yzjo with exactly the same VH CSS... This is an odd bug... Please investigate further. Re-test this before investigating to check it's still an issue.

Test page from initial investigations: http://2bitstudio.co.uk/code-tests/world-service/brand-voice-control-test.html

**To Reproduce**
1. Open Safari and go to https://www.test.bbc.com/pidgin/media-43768328?renderer_env=live
2. Say 'Tap BBC News Pidgin'
3. The link is not activated
4. Say 'Show names' 
5. The link is not listed
6. Say 'Show numbers'
7. The link is not numbered

**Expected behaviour**
1. Go to https://www.test.bbc.com/pidgin/media-43768328?renderer_env=live
2. Say 'Tap BBC News Pidgin'
3. The link should be activated
4. Say 'Show names' 
5. The link should be listed
6. Say 'Show numbers'
7. The link should be numbered

**Testing notes**

- [x] This bug fix is expected to need manual testing.

**Comment by simonsinclair** [comment]
Closing. See: https://github.com/bbc/psammead/pull/3131#issuecomment-592477213.
